kolibri/core/content/test/test_content_app.py

```diff
@@ -553,6 +553,10 @@ class ContentNodeAPITestCase(ContentNodeAPIBase, APITestCase):
     Testcase for content API methods
     """
 
+    def setUp(self):
+        super(ContentNodeAPITestCase, self).setUp()
+        self.maxDiff = 10000
```

Comment:
I think you can also just set this as a class level property:

```
maxDiff = 10000
```

or can set to `None` to give the whole output.

Reply:
ah! sounds good. I thought it peculiar that it said `self.maxDiff` explicitly, but I'll do this